src/timeout.js

```diff
@@ -19,8 +20,7 @@ class Timeout {
 
 	rejected (p) {
 		clearTimeout(this.timer)
-		this.promise._become(p)
-		return false
+		return super.rejected(p)
```

Comment:
I don't know how buble handles `super` (have to go look at the generated code), but is it also worth using `super.fulfilled(p)` above?